wizelineacademy/academy-go-q12021
# [first-deliverable] Add functionality to read pokemons from file

## Description
This PR adds the functionality to read pokemons from the file `pokemon.csv` located at the root of the project. The file structure is as follows:
- Column 1: Pokemon ID
- Column 2: Name
- Column 3: Main type
- Column 3: Subtype
- Column 4: Total score
- Column 5: HP
- Column 6: Attack
- Column 7: Defense
- Column 8: Sp. Atk
- Column 9: Sp. Def
- Column 10: Speed
- Column 11: Generation
- Column 12: Legendary

## How to test
To run this PR, follow the next steps:
1. Install the command by running: `go install github.com/AlejandroSeguraWIZ/academy-go-q12021`
2. Run the server by executing: `academy-go-q12021`
3. Issue a request: 
```
curl -i -X GET 'localhost:8081/pokemons' \              
--header 'Content-Type: application/json'
```
4. You should get a `200 OK` Status and a list with all the pokemons that are contained in the file.
